node/init_test.go

```diff
@@ -25,3 +25,59 @@ func TestInitLight(t *testing.T) {
 	ok := IsInit(dir, Light)
 	assert.True(t, ok)
 }
+
+func TestInitErrForInvalidPath(t *testing.T) {
+	path := "/invalid_path"
+	errLight := Init(path, Light)
+	errFull := Init(path, Full)
+
+	require.Error(t, errLight)
+	require.Error(t, errFull)
+}
+
+func TestInitErrForLockedDir(t *testing.T) {
+	dir := t.TempDir()
+	flock, err := fslock.Lock(lockPath(dir))
+	require.NoError(t, err)
+	defer flock.Unlock() //nolint:errcheck
+
+	errLight := Init(dir, Light)
+	errFull := Init(dir, Full)
+
+	require.Error(t, errLight)
+	require.Error(t, errFull)
+}
+
+func TestIsInitWithBrokenConfig(t *testing.T) {
```

Comment:
it will be beneficial to structure the tests to be readable and easy-to-follow by context of what we are testing. 
I propose to give the a try for groups of test like path/config/dir